src/main/kotlin/minesweeper/domain/coordinate/BaseCoordinateSystem.kt

```diff
@@ -0,0 +1,25 @@
+package minesweeper.domain.coordinate
+
+import minesweeper.common.value.CoordinateValue
+import minesweeper.domain.Position
+
+class BaseCoordinateSystem(
+    height: Int,
+    width: Int
+) : CoordinateSystem {
+    override val coordinate: List<Position>
+        get() = field.toList()
+
+    init {
+        val list = mutableListOf<Position>()
+        for (y in 0 until height) {
+            for (x in 0 until width) {
+                list.add(Position(x = CoordinateValue(value = x), y = CoordinateValue(value = y)))
+            }
+        }
+        this.coordinate = list.toList()
+    }
```

Comment:
아래처럼 buildList를 활용해보아도 좋을 것 같네요.

```suggestion
    override val coordinate: List<Position> = buildList<Position> { /** **/ }
```

Reply:
[d6f78c7](https://github.com/next-step/kotlin-minesweeper/pull/204/commits/d6f78c719624dd2014d350074755f396db34904c)를 사용해서 변경해보았습니다 !!